polkadot/node/service/Cargo.toml

```
@@ -142,6 +142,10 @@ polkadot-node-core-pvf-checker = { path = "../core/pvf-checker", optional = true
 polkadot-node-core-runtime-api = { path = "../core/runtime-api", optional = true }
 polkadot-statement-distribution = { path = "../network/statement-distribution", optional = true }
 
+xcm = { package = "staging-xcm", path = "../../xcm", default-features = false }
+xcm-builder = { package = "staging-xcm-builder", path = "../../xcm/xcm-builder", default-features = false }
+xcm-fee-payment-runtime-api = { path = "../../xcm/xcm-fee-payment-runtime-api" }
```

```suggestion
xcm = { package = "staging-xcm", path = "../../xcm" }
xcm-builder = { package = "staging-xcm-builder", path = "../../xcm/xcm-builder" }
xcm-fee-payment-runtime-api = { path = "../../xcm/xcm-fee-payment-runtime-api" }
```

We need the default `std` feature here